lbz/request.py

```diff
@@ -69,14 +68,13 @@ def json_body(self) -> Optional[dict]:
         if content_type is None:
             return None
         if content_type.startswith("application/json"):
-            if isinstance(self._body, dict):
-                return self._body
-            if self._json_body is None:
+            if not self._json_body:
```

Comment:
I think comparing to `None` makes more sense (previous version), because it can be set to an empty dict - thanks to that this operation won't be repeated every time when someone asks for `request.json_body()` 😉